content/chainguard/chainguard-images/getting-started/wordpress.md

```diff
@@ -0,0 +1,305 @@
+RUN chown -R wordpress:wordpress /var/www/html
+
+USER wordpress
+```
+
+In the `docker-compose.yaml` file, we'll reference the custom Dockerfile and pass the UID as a build argument:
+
+```yaml
+services:
+  app:
+    image: wordpress-local-dev
+    build:
+      context: .
+      dockerfile: Dockerfile
+      args:
+        UID: 1000
+    user: wordpress
+    restart: unless-stopped
+    environment:
+      WORDPRESS_DB_HOST: mariadb
+      WORDPRESS_DB_USER: $WORDPRESS_DB_USER
+      WORDPRESS_DB_PASSWORD: $WORDPRESS_DB_PASSWORD
+      WORDPRESS_DB_NAME: $WORDPRESS_DB_NAME
+    volumes:
+      - ./wp-content:/var/www/html/wp-content
+      - document-root:/var/www/html
+
+  nginx:
+    image: cgr.dev/chainguard/nginx
+    restart: unless-stopped
+    ports:
+      - 8000:8080
+    volumes:
+      - document-root:/var/www/html
+      - ./nginx.conf:/etc/nginx/nginx.conf
+
+  mariadb:
+    image: cgr.dev/chainguard/mariadb
+    restart: unless-stopped
+    environment:
+      MARIADB_ALLOW_EMPTY_ROOT_PASSWORD: 1
+      MARIADB_USER: $WORDPRESS_DB_USER
+      MARIADB_PASSWORD: $WORDPRESS_DB_PASSWORD
+      MARIADB_DATABASE: $WORDPRESS_DB_NAME
+    ports:
+      - 3306:3306
+
+volumes:
+  document-root:
+```
+
+Only the `app` service has changed in this example. We've added a `build` section that references the custom Dockerfile, and we've set the `UID` build argument to `1000` by default. This can be overwritten at runtime when you call `docker compose up`. We've also added a volume share to persist the contents of the `wp-content` folder to the host machine.
+
+To build your custom image and pass along your own UID as build argument, run:
+
+```shell
+docker-compose build --build-arg UID=$(id -u) app
+```
+
+You should get output indicating that the image was successfully built. Now you can get your environment up with:
+
+```shell
+docker-compose up
+```
+
+Once the environment is up and running, you can access your WordPress installation from your browser at `localhost:8000`.
+
+If you go to another terminal window and check the contents of the `02-customizing/wp-content` folder, you'll notice that it was populated with the default WordPress themes and plugins:
+
+```shell
+❯ ls -la wp-content
+total 24
+drwxrwxr-x  4 erika erika 4096 Jul 18 21:16 .
+drwxrwxr-x  3 erika erika 4096 Jul 18 21:15 ..
+-rw-rw-r--  1 erika erika   14 Jul 18 21:05 .gitignore
+-rw-r--r--  1 erika 65533   28 Jan  1  1970 index.php
+drwxr-xr-x  2 erika 65533 4096 Jul 18 21:16 plugins
+drwxr-xr-x 16 erika 65533 4096 Jul 18 21:16 themes
+```
+
+This is only possible because of the custom Dockerfile we created, which added a `wordpress` user with the same UID as the local user on the host machine.
+
+You can now install new themes and plugins, and they will persist between container rebuilds.
+
+To stop the services, type `CTRL+C` in the terminal where the services are running, and then run:
+
+```bash
+docker-compose down
```

Comment:
```suggestion
docker compose down
```

Reply:
This was probably a wrong autocomplete from my editor! supposed to be `docker compose`